src/Playwright.Tests/BaseTests/SetUpFixture.cs

```diff
@@ -0,0 +1,46 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using Microsoft.Extensions.Logging;
+using Microsoft.Playwright.NUnit;
+using Microsoft.Playwright.NUnit.Logging;
+using NUnit.Framework;
+
+namespace Microsoft.Playwright.Tests
+{
+    [SetUpFixture]
+    class SetUpFixture
+    {
+        [OneTimeSetUp]
+        public void ConfigurePlaywrightTests()
+        {
+            // Use Default Logging
+            PlaywrightTest.LoggerFactory = Extensions.Logging.LoggerFactory.Create(builder => builder
+                   .ClearProviders()
+                   // To debug Playwright for .NET, we can set this to Trace
+                   .SetMinimumLevel(LogLevel.Debug)
```

Comment:
I'm leaving this here, because it'll come in handy in the future. But this is technically not needed.